yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/MediaValue.java

```diff
@@ -0,0 +1,32 @@
+package com.yoti.api.client.spi.remote;
+
+import static org.bouncycastle.util.encoders.Base64.toBase64String;
+
+import com.yoti.api.client.Media;
+
+public class MediaValue implements Media {
+
+    private final String mimeType;
+    private final byte[] content;
+
+    public MediaValue(String mimeType, byte[] content) {
+        this.mimeType = mimeType;
+        this.content = content;
+    }
+
+    @Override
+    public String getMimeType() {
+        return mimeType;
+    }
+
+    @Override
+    public byte[] getContent() {
+        return content.clone();
+    }
+
+    @Override
+    public String getBase64Content() {
+        return String.format("data:%s;base64,%s", getMimeType(), toBase64String(getContent()));
+    }
+
+}
```

Comment:
Looks like this file is almost identical to ImageAttributeValue. Should we have `JpegAttributeValue` and `PngAttributeValue` extend from this, and deprecate ImageAttributeValue?

Reply:
Oh ok, how come it broke the unit and integration tests? I would have thought if the exposed methods are exactly the same, it would be ok